### PR TITLE
DP for AdMob rename

### PR DESCRIPTION
**Pull request by standa-dev**

### CODE REVIEW

File: app/src/main/kotlin/Navigation.kt [hunk 1]
```diff
@@ -64,8 +65,14 @@ val screens = mutableMapOf(
             "Dynamic Price Banner + Video",
             "Dynamic Price Inline Video",
             "Dynamic Price Interstitial",
+        )),
+        NavigationAdapter(destination = "AdMob Dynamic Price", header = "AdMob", items = arrayOf(
```
Comment (brugg) [inline app/src/main/kotlin/Navigation.kt:69]: We should not references to `AdMob Dynamic Price` in code. Please rename it to `Dynamic Adapters for AdMob`

File: app/build.gradle.kts [hunk 2]
```diff
@@ -74,7 +79,11 @@ dependencies {
 
     /* Google Mediation Adapters and Dynamic Price */
     api("com.adsbynimbus.android:extension-google:$nimbusVersion")
-    api("com.google.android.gms:play-services-ads:22.1.0")
+    api("com.google.android.gms:play-services-ads:22.2.0")
+
+    /* Admob and Dynamic Price */
```
Comment (brugg) [inline app/build.gradle.kts:84]: Please rename it to `Dynamic Adapters for AdMob`

File: app/src/main/kotlin/Navigation.kt (hunk 1 above)
Comment (standa-dev) [reply app/src/main/kotlin/Navigation.kt:69]: addressed in https://github.com/timehop/nimbus-android-sample/pull/52

File: app/build.gradle.kts (hunk 2 above)
Comment (standa-dev) [reply app/build.gradle.kts:84]: addressed in https://github.com/timehop/nimbus-android-sample/pull/52

**Comment by standa-dev** [comment]
Closing in favor of https://github.com/timehop/nimbus-android-sample/pull/52

**Pull request closed without merging**